lib/chef/provider/aws_route_table.rb

```diff
@@ -89,6 +91,30 @@ def update_routes(vpc, route_table)
     end
   end
 
+  def update_virtual_private_gateways(route_table, gateway_ids)
+    current_propagating_vgw_set = route_table.client.describe_route_tables(route_table_ids: [route_table.id]).route_table_set.first.propagating_vgw_set
```

Comment:
Not according to their API docs, but they could be lacking. I'll try it and see.